src/lib/triggers.js

```diff
@@ -9,6 +9,14 @@ import { normalizeAgent } from './api';
 
 const agentCacheExpiry = 3600000;
 let agentPromise;
+
+window.mobilecheck = function() {
```

Comment:
Please, move this code/function to the `helper` file.

Reply:
By helpers did you mean helpers.stories.js?
Or should I create a new file `scr/helpers.js` as there is no helpers.js file

Reply:
done (y)